data/korath ships.txt

```diff
@@ -1025,10 +1025,10 @@ ship "Model 64"
 	engine -19 159
 	engine 19 159
 	gun 0 -159 "Korath Minelayer"
-	turret -35 -69 "Korath Disruptor"
 	turret 20 -74 "Korath Slicer Turret"
-	turret 34 77 "Korath Disruptor"
 	turret -29 71 "Korath Slicer Turret"
+	turret -35 -69 "Korath Disruptor"
```

Comment:
saw this line needs to move up a line, but I covered that in my PR that's upcoming